spec/rails_helper.rb

```diff
@@ -20,12 +20,17 @@
 require 'rspec/core'
 require 'spec_helper'
 require 'rspec/rails'
+require 'factory_bot_rails'
+require 'webmock/rspec'
+require 'shoulda/matchers'
+require 'pundit/rspec'
+Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |file| require file }
 
 ActiveRecord::Migration.maintain_test_schema!
 WebMock.disable_net_connect!(allow_localhost: true)
 
 RSpec.configure do |config|
-  config.include Devise::Test::ControllerHelpers, type: :controller
+  config.include Devise::Test::IntegrationHelpers
```

Comment:
Just curious, what's the difference?

Reply:
I think the other one is just for controller testing and this one works for system/feature testing

Reply:
Just asking because I think this might slow tests down. Related, how cool would it be to have parallel testing OOB 😉